guides/source/active_record_migrations.md

```diff
@@ -967,36 +959,34 @@ end
 
 In many ways this is exactly what it is. This file is created by inspecting the
 database and expressing its structure using `create_table`, `add_index`, and so
-on. Because this is database-independent, it could be loaded into any database
-that Active Record supports. This could be very useful if you were to
-distribute an application that is able to run against multiple databases.
-
-NOTE: `db/schema.rb` cannot express database specific items such as triggers,
-sequences, stored procedures or check constraints, etc. Please note that while
-custom SQL statements can be run in migrations, these statements cannot be reconstituted
-by the schema dumper. If you are using features like this, then you
-should set the schema format to `:sql`.
-
-Instead of using Active Record's schema dumper, the database's structure will
-be dumped using a tool specific to the database (via the `db:structure:dump`
-rails task) into `db/structure.sql`. For example, for PostgreSQL, the `pg_dump`
-utility is used. For MySQL and MariaDB, this file will contain the output of
-`SHOW CREATE TABLE` for the various tables.
-
-Loading these schemas is simply a question of executing the SQL statements they
-contain. By definition, this will create a perfect copy of the database's
-structure. Using the `:sql` schema format will, however, prevent loading the
-schema into a RDBMS other than the one used to create it.
+on.
+
+`db/schema.rb` cannot express everything your database may support such as
+triggers, sequences, stored procedures or check constraints, etc. Please note
+that while custom SQL statements can be run in migrations, these statements
+cannot be reconstituted by the schema dumper. If you are using features like
+this, you can set the schema format to `:sql` or you can use `db/schema.rb`
+solely as a reference. In this case you would create a new instance of your
+database by running your migrations from scratch instead of running `rails
+db:schema:load`
+
+When the schema format is set to `:sql`, the database structure will be dumped
+using a tool specific to the database into `db/structure.sql`. For example, for
+PostgreSQL, the `pg_dump` utility is used. For MySQL and MariaDB, this file will
+contain the output of `SHOW CREATE TABLE` for the various tables.
+
+Loading these `db/structure.sql` is done by executing the SQL statements it
+contains. By definition, this will create a perfect copy of the database's
+structure. However, `db/structure.sql` tends to be harder to use as a reference
```

Comment:
Why "harder to use as a reference" matters here? If you need to you, you will have to use it.

Reply:
I can see leaving this out entirely. 

Our thinking was that many may be perfectly happy to live in a world where they *don't* use `schema.rb` to generate a schema but *do* use it for reference, keeping schema format set to `:ruby`

Reply:
> live in a world where they don't use schema.rb to generate a schema

This world should not exist. The schema.rb and structure.sql is the only way to generate a schema. Migrations are viewed as ephemeral and disposable. Some applications even delete all migrations after they are run in production. We even discussed doing that by default in Rails.